protos/fpc/fpc.proto

```diff
@@ -93,14 +93,22 @@ message InitEnclaveMessage {
 message CleartextChaincodeRequest {
     // the function and args to invoke
     protos.ChaincodeInput input = 1;
-
-    // key to (aes128-gcm) encrypt ChaincodeResponse
-    bytes return_encryption_key = 3;
 }
 
 message ChaincodeRequestMessage {
-    // an RSA-encryption of the serialization of CleartextChaincodeRequest with the chaincode encryption key
+    // an encryption (symmetric) of the serialization of CleartextChaincodeRequest with KeyTransportMessage.request_encryption_key
     bytes encrypted_request = 1;
+
+    // an encryption (asymmetric) of the serialization of request KeyTransportMessage with AttestedData.chaincode_ek
+    bytes encrypted_key_transport_message = 2;
+}
+
+message KeyTransportMessage {
+    // key to decrypt CleartextChaincodeRequest
+    bytes request_encryption_key = 1;
+
+    // key to encrypt CleartextChaincodeResponse
```

Comment:
worth mentioning in comment what type of key it is? (AES-GCM-128?)

Reply:
I removed the details about RSA and AES keys as they may change with whatever crypto lib is used (even though I don't see this changed soon).